src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -14,9 +14,11 @@ import Switch from '@components/Switch';
 import tagPropTypes from '@components/tagPropTypes';
 import Text from '@components/Text';
 import transactionPropTypes from '@components/transactionPropTypes';
+import ViolationMessages from '@components/ViolationMessages';
 import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsPropTypes} from '@components/withCurrentUserPersonalDetails';
 import useLocalize from '@hooks/useLocalize';
 import usePermissions from '@hooks/usePermissions';
+import useViolations from '@hooks/useViolations';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
```

Comment:
```suggestion
import useThemeStyles from '@hooks/useThemeStyles';
import useViolations from '@hooks/useViolations';
```

Reply:
This looks like it's still open, but the preview is wrong if you compare with the current file